dockers/docker-ptf/Dockerfile.j2

```diff
@@ -96,8 +96,11 @@ RUN mkdir /var/run/sshd \
  && sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config \
  && sed -i '$aUseDNS no' /etc/ssh/sshd_config
 
-COPY ["supervisord.conf", "sshd.conf", "ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+COPY ["supervisord.conf", "/etc/supervisor/"]
+COPY ["conf.d/supervisord.conf", "conf.d/sshd.conf", "conf.d/ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+
+RUN mkdir -p /var/log/supervisor
 
 EXPOSE 22
 
-ENTRYPOINT ["/usr/bin/supervisord"]
+ENTRYPOINT ["/usr/local/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
```

Comment:
I see. Just note that in specifying the config file, supervisor is not loading /etc/supervisor.conf, which you modified below. None of the configuration in that file will be loaded. If you need config from both, you'll either have to let the main file include the conf.d file, or copy all pertinent config into the conf.d file, in which case, you should delete the main file to prevent confusion. Personally, I'm OK with the warning message, at least for now.